README.md

```diff
@@ -1,2 +1,22 @@
 # ensembl-rest
-rest container for ensembl services
+Language agnostic RESTful data access to Ensembl data over HTTP
+Inspired by andrewyatz's vagrant effort https://github.com/andrewyatz/vagrant_machines/tree/master/ensembl/rest
+## pre-requisites
+docker
+## setup perl environment, ubuntu packages, etc.
+docker build -t ensembl-base -f Dockerfile-base . 
+## configure it
+docker build -t ensembl-rest -f Dockerfile-rest  . 
+## start it 
+docker run -d  --name ensembl-rest -p 3000:3000  -it ensembl-rest
```

Comment:
Should we publish this to ohsucompbio/ensemble-rest?

Reply:
Sounds like a plan